### PR TITLE
Add Pomodoro and Pomodoro key binding to tmux

### PR DESCRIPTION
**Pull request by tlball**
Added the [Thyme](https://github.com/hughbien/thyme) gem for
Pomodoros with integration into tmux. Also added a key-binding for this.

Key Bindings:
Ctrl+A t
After prompt, enter the preferred option (displayed in the prompt)
p for standard Pomodoro
b for short break
t for long break

To stop the current Pomodoro, run any of the above options.
The timer will show in the status bar on the right side.

### CODE REVIEW

**Comment by jhirbour** [comment]
I <3 ! It's probably good to note the "install" steps for this. Off the top of my head:

```
cd ~/.homesick/repos/dotfiles/
bundle
homesick link dotfiles
```

File: home/.thymerc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run
+end
+
+option :s, 'seconds num', 'run with custom seconds' do |num|
+  @timer = num.to_i
+  run
+end
+
+# before do
+#   `mplayer ~/music/flight-of-the-bumble-bee.mp3 &`
```
Comment (treznick) [inline home/.thymerc:33]: This is awesome

Comment (jnimety) [inline home/.thymerc:33]: :ship: it!

On Wed, May 13, 2015 at 10:50 AM, Tom Reznick notifications@github.com
wrote:

> In home/.thymerc
> https://github.com/ContinuityControl/dotfiles/pull/16#discussion_r30238083
> :
> 
> > -  set :timer, 5*60
> > -  run
> >   +end
> >   +
> >   +option :l, :long_break, 'start a long break' do
> > -  set :timer, 10*60
> > -  run
> >   +end
> >   +
> >   +option :s, 'seconds num', 'run with custom seconds' do |num|
> > -  @timer = num.to_i
> > -  run
> >   +end
> >   +
> >   +# before do
> >   +#   `mplayer ~/music/flight-of-the-bumble-bee.mp3 &`
> 
> This is awesome
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ContinuityControl/dotfiles/pull/16/files#r30238083.

## 

Joel Nimety
Senior VP, Engineering
Continuity
203.285.8133 (office)
203.763.9263 (cell)
http://www.continuity.net
http://www.linkedin.com/in/jnimety

## 

## 

---

This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.

**Comment by tlball** [comment]
:ship:ing it. 

Also, thanks @jhirbour for the install steps reminder. A 'bundle' is definitely required. I think these are all also mentioned in the updates sections of the README doc for the dotfiles.

**Comment by caseywatts** [comment]
Zach and I found a bug maybe related to this?
image

Somehow nothing is appearing in the bottom right for us after setting a pom, our names are gone too.

**Comment by jhirbour** [comment]
@caseywatts did you make sure you weren't scrolled up/down?

**Comment by tlball** [comment]
@casewatts, Any good way to reproduce this? Or is it just something you ran across?

> On May 20, 2015, at 11:40 AM, Casey Watts notifications@github.com wrote:
> 
> Zach and I found a bug maybe related to this?
>  https://cloud.githubusercontent.com/assets/273653/7731279/5a83c494-feed-11e4-9b92-9ca7edc5f654.png
> Somehow nothing is appearing in the bottom right for us after setting a pom, our names are gone too.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/ContinuityControl/dotfiles/pull/16#issuecomment-103953238.

## 

## 

---

This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.

**Comment by caseywatts** [comment]
I'm not sure how to reproduce it sorry, I just wanted to record that it did
happen.

On Wed, May 20, 2015 at 5:42 PM, Tricia Ball notifications@github.com
wrote:

> @casewatts, Any good way to reproduce this? Or is it just something you
> ran across?
> 
> > On May 20, 2015, at 11:40 AM, Casey Watts notifications@github.com
> > wrote:
> > 
> > Zach and I found a bug maybe related to this?
> > <
> > https://cloud.githubusercontent.com/assets/273653/7731279/5a83c494-feed-11e4-9b92-9ca7edc5f654.png
> > 
> > Somehow nothing is appearing in the bottom right for us after setting a
> > pom, our names are gone too.
> > 
> > —
> > Reply to this email directly or view it on GitHub <
> > https://github.com/ContinuityControl/dotfiles/pull/16#issuecomment-103953238
> > .
> 
> ## 
> 
> ## 
> 
> ---
> 
> This email and all files transmitted with it are intended solely for use
> by
> the intended recipient. Any other use is prohibited. If you received this
> email in error, please notify the sender, delete the email and destroy all
> attachments. Opinions and ideas expressed herein are solely those of the
> sender, and may not reflect the official position of Continuity.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ContinuityControl/dotfiles/pull/16#issuecomment-104047774
> .

## 

## 

---

This email and all files transmitted with it are intended solely for use by 
the intended recipient.  Any other use is prohibited.  If you received this 
email in error, please notify the sender, delete the email and destroy all 
attachments.  Opinions and ideas expressed herein are solely those of the 
sender, and may not reflect the official position of Continuity.

**Pull request merged**